cmd/gateway-azure.go

```diff
@@ -144,6 +144,8 @@ func azureToObjectError(err error, params ...string) error {
 		err = BucketNameInvalid{Bucket: bucket}
 	case "RequestBodyTooLarge":
 		err = PartTooBig{}
+	case "InvalidMetadata":
+		err = MetadataTooLarge{}
```

Comment:
`InvalidMetadata -> MetadataTooLarge` will be an incorrect translation. Better introduce new error type as no existing errors are suitable for `InvalidMetadata` error.